utils/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_executable(nemsio_read nemsio_read.f90)
+target_link_libraries(nemsio_read nemsio bacio::bacio_4 w3nco::w3nco_d)
```

Comment:
on second thought, `bacio` and `w3nco` are not used in these utilities. Don't need them here.